### DISCUSSION TITLE
Charts not neatly generated. Some charts are cropped off.

### DISCUSSION THREAD

**Discussion by shwetabhattad-TU** [sinaptik-ai/pandas-ai#615] [General]
Charts not neatly generated. Some charts are cropped off.

![cropped_image](https://github.com/gventuri/pandas-ai/assets/135302428/cc65d03e-a1bf-4f2e-a571-d9b80990dfc4)

**Comment by adityamiskin** [comment]
> Charts not neatly generated. Some charts are cropped off.
> 
> ![cropped_image](https://user-images.githubusercontent.com/135302428/272501949-cc65d03e-a1bf-4f2e-a571-d9b80990dfc4.png)

You can tell the llm to use bbox_inches="tight" while saving the image with plt.savefig().

**Reply by shwetabhattad-TU** [reply]
This is not working. Tried it.

**Reply by shwetabhattad-TU** [reply]
Using pandasai 1.3.2. Any updates on this.
Sometimes even the xaxis values are getting overlapped and sometimes the legend is overlapping on the chart.
